frontend/scripts/router/route-subscriber.js

```diff
@@ -20,6 +21,7 @@ export default function routeSubscriber(store) {
 
     onRouteChange({ routesMap, type } = {}) {
       const filename = routesMap[type].page;
+      updateScroll();
```

Comment:
I don't like this. If the query params change and the page didn't it will scroll to top.

Reply:
I thought so too, but no. Tested on the sankey, but will retest again tomorrow.